docs/book/src/reference/jobs.md

```diff
@@ -0,0 +1,79 @@
+# Jobs
+
+This document intents to provide an overview over our jobs running via Prow, GitHub actions and Google Cloud Build.
+
+## Builds and Tests running on the default branch (currently v1alpha4)
+
+> NOTE: To see which test jobs execute which tests or e2e tests, you can click on the links which lead to the respective test overviews in testgrid.
+
+### Presubmits
+
+Prow Presubmits:
+* [pull-cluster-api-build-main] ./scripts/ci-build.sh
+* \[opt\] \[always-run:false\] [pull-cluster-api-make-main] ./scripts/ci-make.sh
```

Comment:
I wonder if it would be easier to read if we take advantage of unicode characters to do something like:

### Legend

✳️ Jobs that don't have to be run successfully for merge
✴️ Jobs that are not triggered automatically for every commit

### Presubmits

Prow Presubmits:
* [pull-cluster-api-build-main] ./scripts/ci-build.sh
* ✳️ ✴️ [pull-cluster-api-make-main] ./scripts/ci-make.sh
* ✳️ [pull-cluster-api-apidiff-main] ./scripts/ci-apidiff.sh
* [pull-cluster-api-verify] ./scripts/ci-verify.sh

Reply:
Good idea, looks a lot better now.